.vscode/launch.json

```diff
@@ -1,17 +1,29 @@
 {
     "version": "0.2.0",
     "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Debug CLI - init all",
+            "cwd": "${workspaceFolder}/vehicle-app-repo",
+            "program": "${workspaceFolder}/bin/dev",
+            "args": ["init"]
+        },
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Debug CLI - init with package and version",
+            "cwd": "${workspaceFolder}/vehicle-app-repo",
+            "program": "${workspaceFolder}/bin/dev",
+            "args": ["init", "-p", "devenv-runtimes@v3.0.0"]
+        },
```

Comment:
I left the changes inside, but of course these could be reverted because if we add launch options for each and every command it could get out of hand